.travis.yml

```diff
@@ -0,0 +1,36 @@
+sudo: required
```

Comment:
I think travis made pg 9.5 available on trusty containers a couple of months ago, dont need this and the before_script anymore, methinks